R/post_ts.R

```diff
@@ -76,7 +74,7 @@ post_ts = function(files, on_exists=c("stop", "skip", "replace", "merge","archiv
             message("deleting timeseries data before replacement: ", ts_id)
             message("posting file to site ", ts_path$site_name, ", timeseries ", ts_path$ts_name)
           }
-          item_replace_files(ts_id, files = basename(files[i]), all=FALSE)
+          item_replace_files(ts_id, files = files[i], all=FALSE)
```

Comment:
wondering about this and line 95: yes, `basename` is wrong b/c removes the extension, but couldn't `files[1]` often include a full file path? maybe it makes more sense to use `ts_path$file_name` here?